poetry.lock

```diff
@@ -40,15 +40,15 @@ name = "appnope"
 version = "0.1.2"
 description = "Disable App Nap on macOS >= 10.9"
 category = "main"
-optional = true
+optional = false
```

Comment:
@naveen521kk : Is it ok when the optional parameter is false?
I don't know why this happend

Reply:
if you ran `poetry lock` or `poetry update` and this happened it should be fine.

Reply:
@Darylgolden : Did you run `poetry lock` or `poetry update` ?

Reply:
I did `poetry add`.

Reply:
This is also fine, but in particular since this is a dependency for jupyter-lab/Colab, it should be placed in the extras

Reply:
Yep, that is possible, but I think it is also ok to be in the main dependencies, no?
I am thinking of the case that someone had already installed jupyter notebook and then installs manim, but has no jupyter lab, it would be nice when this cell magic works out of the box. Or are the extras automatically installed with `pip install manim`? If it would work automatically, then we can add it to the extras.
Further, I have a lot of extra installed packages in my manim environment, and I am afraight that when I type `poetry update` that these will be also added to the poetry file.

Reply:
No need to worry about extra installed packages in your manim environment. `poetry update` will only update the versions of the packages listed in pyproject.toml.

The extras are not automatically installed via pip, nor are they by poetry.
In either package manager, you must specify the extra e.g.:
`poetry install -E jupyterlab`
`pip install manim[jupyterlab]`

Since jupyterlab is not a requirement and is already part of the extras, its dependencies should also be extras.